migrations/20161104170802-sample-remove-name-index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    queryInterface.removeIndex('Samples', 'SampleName');
+  },
+
+  down: function (queryInterface, Sequelize) {
+    queryInterface.addIndex(
+      'Samples',
+      ['name'],
+      {
```

Comment:
https://devcenter.heroku.com/articles/expensive-queries. Looks like unused indexes could also be a cause of slow DML operations.